src/components/FileViewer.jsx

```diff
@@ -0,0 +1,38 @@
+/* global cozy */
+
+import utilStyles from '../styles/utils'
+import React from 'react'
+import Loading from './Loading'
+
+class FileViewer extends React.Component {
```

Comment:
You're right, I am thinking of `IntentHandler` since it is also used to handle the `GET_URL` intent. What do you think ?